docs/openstack/edpm_adoption.md

```diff
@@ -305,3 +294,179 @@ done
     ```
     oc wait --for condition=Ready osdpns/openstack --timeout=30m
     ```
+
+  ## Nova compute services fast-forward upgrade from Wallaby to Antelope
+
+> **NOTE**: Nova compute services upgrade cannot be performed in a lock-step with
+> Nova control plane services as those are managed independently by EDPM
+> ansible, and Kubernetes operators. The steps below implement that FFU
+> coordination instead, becaue there is yet a signalling interface between
+> different control loops that EDPM ansible and OCP operators utilize.
+
+* Define Nova compute services FFU config snippet for EDPM:
+    ```yaml
+    oc apply -f - <<EOF
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: nova-compute-ffu
+      namespace: openstack
+    data:
+      05-nova-ffu-extra.conf: |
+        [upgrade_levels]
+        compute=auto
+    EOF
+    ```
+* Patch EDPM nodeset for new Nova compute FFU configration service:
+    ```yaml
+    oc patch osdpns/openstack -n openstack --type=merge --patch '
+    spec:
+      services:
+        - nova-compute-ffu
+    '
+    ```
+
+* Upgrade Nova compute services on EDPM:
+    ```yaml
+    oc apply -f - <<EOF
+    apiVersion: dataplane.openstack.org/v1beta1
+    kind: OpenStackDataPlaneService
+    metadata:
+      name: nova-compute-ffu
+      label: nova.compute.ffu
+      namespace: openstack
+    spec:
+      configMaps:
+        - nova-compute-ffu
+      playbook: osp.edpm.nova
+    ---
+    apiVersion: dataplane.openstack.org/v1beta1
+    kind: OpenStackDataPlaneDeployment
+    metadata:
+      name: openstack-nova-compute-ffu
+      namespace: openstack
+    spec:
+      nodeSets:
+        - openstack
+      servicesOverride:
+        - nova-compute-ffu
+    EOF
+    ```
+  Here we limited ansible execution to `nova-compute-ffu`, which calls the
+  `osp.edpm.nova` playbook.
+
+  > **NOTE**: In production environments, Nova compute services upgrading should
+  > be done in chunks. For each chunk of compute hosts, create a pair of
+  > `OpenStackDataPlaneNodeSet` and `OpenStackDataPlaneDeployment` CRs, which
+  > results in ansible batched executions.
+
+* Wait for Nova compute FFU dataplane service deployed:
+    ```bash
+    oc wait --for condition=Ready osdpns/openstack --timeout=5m
+    oc wait --for condition=Ready osdpd/openstack-nova-compute-ffu --timeout=5m
+    ```
+
+* Enable upgrade levels calculations for Nova conductors control plane:
```

Comment:
no longer needed for the control plane neither, after https://github.com/openstack-k8s-operators/nova-operator/pull/578